projects/admin/src/app/scss/styles.scss

```diff
@@ -74,6 +74,12 @@ header {
   content: " \f08e";
 }
 
+span.no-data {
```

Comment:
Perhaps we need to add a general prefix class for all css global application classes.

Reply:
I will ask the devs feeling the about this question at a daily meeting